### DISCUSSION TITLE
Unable to Install bpftrace on Kali Linux

### DISCUSSION THREAD

**Discussion by carbonvibes** [bpftrace/bpftrace#3052] [Q&A]
<img width="1131" alt="Screenshot 2024-03-13 at 12 45 42 AM" src="https://github.com/bpftrace/bpftrace/assets/116697912/772cc25d-5b51-44e0-ac9b-1977c7e54db4">

I used bpftrace the previous week, and it was completely working fine, but now when I try to install bpftrace in Kali (New VM instance) it gives me this error. I've enabled the kali experimental repo source as well but still no luck. Also I tried to look up the package using apt-cache as well but it shows no entry for bpftrace.

**Comment by danobi** [comment]
Hi, this looks like a packaging issue. I recommend filing a bug against kali -- I don't believe folks here are involved with kali

**Comment by Engg-blogger** [comment]
I am facing exact similar issue in ubuntu-20 as well. Just started happening this week.

**Reply by carbonvibes** [reply]
I just found out that Kali officially removed bpftrace PACKAGE from Kali-rolling about a week ago. I guess probably the same case with Ubuntu, pls check the Ubuntu package manager website for more info.

<img width="1195" alt="Screenshot 2024-03-13 at 5 28 43 AM" src="https://github.com/bpftrace/bpftrace/assets/116697912/4bbf8cd5-cb4b-43f6-9141-c0ca372b8cdf">
